examples/hybrid/driver.jl

```diff
@@ -136,7 +137,7 @@ function additional_cache(Y, params, model_spec, dt; use_tempest_mode = false)
             )
         ),
         (; Δt = dt),
-        (; enable_default_remaining_tendency = isnothing(turbconv_model)),
+        (; enable_anelastic_dycore = anelastic_dycore),
```

Comment:
```suggestion
        (; enable_anelastic_dycore = model_spec.anelastic_dycore),
```
Can we avoid the closure here and store this in `model_spec`?